SPECS/kernel/config_aarch64

```diff
@@ -8777,10 +8776,8 @@ CONFIG_COMMON_CLK_MT8192=y
 # CONFIG_COMMON_CLK_MT8192_VENCSYS is not set
 CONFIG_COMMON_CLK_MT8195=y
 CONFIG_COMMON_CLK_MT8195_APUSYS=y
-CONFIG_COMMON_CLK_MT8195_AUDSYS=y
```

Comment:
result of microsoft/CBL-Mariner-Linux-Kernel@3d02397

This config does nothing in the source